src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -166,18 +180,7 @@ export default class OverflowMenu extends Component {
             description={iconDescription}
             style={{ width: '100%' }}
           />
-          {!open ? null : floatingMenu ? (
-            <FloatingMenu
-              menuPosition={this.state.menuPosition}
-              menuDirection="bottom"
-              menuOffset={flipped ? menuOffsetFlip : menuOffset}>
-              <ul className={overflowMenuOptionsClasses}>
-                {childrenWithProps}
-              </ul>
-            </FloatingMenu>
-          ) : (
-            <ul className={overflowMenuOptionsClasses}>{childrenWithProps}</ul>
-          )}
+          {!open ? null : wrappedMenuBody}
```

Comment:
Is the following equivalent:

```
{open && wrapperMenuBody}
```

Reply:
@joshblack If React stop rendering for any falsy values, yes. (Was unsure about that)